src/css/styles.scss

```diff
@@ -164,6 +164,11 @@
     box-shadow: 0 0 0 2px var(--focus-highlight-color);
   }
 
+  a:focus-visible {
+    outline: none;
+    text-decoration: underline;
+  }
```

Comment:
I still think `outline` is better to perceive when it comes to Accessibility so instead I would suggest to make outline styling better and consistent across all browsers eg in mac its looks like 👇
<img width="551" alt="Screenshot 2023-05-11 at 12 27 56 PM" src="https://github.com/excalidraw/excalidraw/assets/11256141/292cccc1-5d05-49ea-a114-4b380ba89179">

Reply:
outline is still not pretty to be shown as the first thing when you open the dialog. So if we go with outline we should stop focusing just any first focusable element on dialog open, only if it's a button.